### ISSUE TITLE
Loader Exceptions after upgrade from 3.0.1

### ISSUE THREAD

**Issue by EcTomBot** [TrakHound/MTConnect.NET#22]
Hi

I updated library from 3.0.1 to 3.2 version and I writed the follow c# code into my existing project:

`
var Client= new MTConnectCurrentClient(@"10.0.0.204:8040", "NH802HB0002");
if (current != null)
{
    var document = Client.Get();
}
Client.OnInternalError += Errore;
`

On Client.Get() one internal error occured with two exceptions (italian language):

Il metodo 'AuthenticateAsync' del tipo 'Microsoft.AspNetCore.Http.Authentication.Internal.DefaultAuthenticationManager' dell'assembly 'Microsoft.AspNetCore.Http, Version=2.2.2.0, Culture=neutral, PublicKeyToken=adb9793829ddae60' non ha un'implementazione.

Non è stato possibile caricare il tipo 'Microsoft.AspNetCore.Http.Features.Authentication.ChallengeBehavior' dall'assembly 'Microsoft.AspNetCore.Http.Features, Version=5.0.15.0, Culture=neutral, PublicKeyToken=adb9793829ddae60'.

If I create a new Visual Studio project the error does not occur.

What happen?
Is there any conflict with the dependencies of the other nuget packages? 

Thanks
Tomaso

**Comment by PatrickRitchie** [comment]
@EcTomBot What .NET framework version are you using? Also, are you using any other Nuget packages that could be causing a conflict?

**Comment by EcTomBot** [comment]
Hi Patrick

I found that the problem occurs by installing the PLCcom.Opc.Ua.Sdk library as in the example project that you find below.
[ConsoleApp2.zip](https://github.com/TrakHound/MTConnect.NET/files/8415356/ConsoleApp2.zip)

I noticed that it also depends on the dependency version of the PLCcom.Opc.Ua.Sdk library.
![PLCcom Opc Ua Sdk](https://user-images.githubusercontent.com/55627306/161688499-36425fe5-c3ed-4921-be36-05d24e1c0e11.PNG)

Do you have a solution to make the two libraries coexist?

Thank you
Tomaso

**Comment by PatrickRitchie** [comment]
@EcTomBot I wasn't able to reproduce the issue you were seeing. I created a new .NET Framework 4.8 console app project and added both nuget packages (MTConnect.NET, and PLCcol.Opc.Ua.Sdk) and I still didn't see any issues.

I noticed in the project you sent, there were a lot of the dependency nuget packages referenced individually (I don't know if that is the way it is on your end but that is how it showed up when I opened the project). I've had issues with that before where nuget packages have conflicting versions causing similar exceptions as the one you received.

Below are screenshots of what I got to work:

![Example-Nuget-Packages](https://user-images.githubusercontent.com/13772118/161892885-48df1551-9b9e-4d7f-8d2d-0a0e4571ed62.png)

![Example-Nuget-Packages-02](https://user-images.githubusercontent.com/13772118/161893052-9b1658a6-f040-465a-a37c-65caacc86a04.png)
